pgjdbc/src/main/java/org/postgresql/core/v3/BatchedQueryDecorator.java

```diff
@@ -151,28 +148,6 @@ boolean isPreparedFor(int[] paramTypes) {
     return isStatementParsed() && super.isPreparedFor(paramTypes);
   }
 
-  @Override
-  byte[] getEncodedStatementName() {
-    if (batchedEncodedName == null) {
-      String n = super.getStatementName();
-      if (n != null) {
-        batchedEncodedName = Utils.encodeUTF8(n);
-      }
-    }
-    return batchedEncodedName;
-  }
-
-  @Override
-  void setStatementName(String statementName) {
-    if (statementName == null) {
-      batchedEncodedName = null;
-      super.setStatementName(null);
-    } else {
-      super.setStatementName(statementName);
-      batchedEncodedName = Utils.encodeUTF8(statementName);
-    }
-  }
```

Comment:
I don't know if it became useless because of my descriptor recycling approach or something else, but it does not seem to be any use as far as I can tell.